masterlist.yaml

```diff
@@ -5120,6 +5126,12 @@ plugins:
         itm: 6
         udr: 0
         nav: 0
+  - name: 'Wasteland Defense.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/38049/?' ]
+    msg:
+      - type: say
+        content: 'Contains wild edits to records 00116185 and 00116187.'
```

Comment:
It's a bit vague, are they supposed to remove those two records?

Reply:
Or just a part of them?

Reply:
The two records should be removed, they're going to break any mod that would interact with Benny or the Tops.

Reply:
You could add that to the message as well.

>Contains wild edits to records 00116185 and 00116187. These two records should be removed, as they're going to break any mod that would interact with Benny or the Tops.

Reply:
Fair enough, I'll commit the changes in a few minutes.

Reply:
Thanks, It will save time in future, because someone is going to ask why otherwise.